config/jobs/kubernetes-sigs/sig-windows/release-1.18-windows.yaml

```diff
@@ -164,13 +164,13 @@ periodics:
       - --aksengine-win-binaries
       - --aksengine-deploy-custom-k8s
       # Specific test args
-      - --test_args=--node-os-distro=windows --ginkgo.focus=\[Conformance\]|\[NodeConformance\]|\[sig-windows\]|\[sig-apps\].CronJob|\[sig-api-machinery\].ResourceQuota --ginkgo.skip=\[LinuxOnly\]|\[Serial\]|\[Slow\]|Guestbook.application.should.create.and.stop.a.working.application
+      - --test_args=--node-os-distro=windows --ginkgo.focus=$(GINKGO_FOCUS_PARALLEL) --ginkgo.skip=$(GINKGO_SKIP_PARALLEL)
```

Comment:
Where is $(GINKGO_FOCUS_PARALLEL) being set?

Reply:
It's set in https://github.com/kubernetes/test-infra/pull/21457/files#diff-df283afbd02f2f70d68a1151d0b93b86f3ce1a8bf4766ef4d812aae58ac26a31R11-R18

Reply:
found it.  The file was minimized by default